src/addon-manager.js

```diff
@@ -386,8 +385,8 @@ class AdapterManager extends EventEmitter {
       return Promise.reject(err);
     }
 
-    // Get any saved settings for this adapter.
-    const key = `adapters.${manifest.name}`;
+    // Get any saved settings for this add-on.
+    const key = `addons.${manifest.name}`;
```

Comment:
Does this need some type of settings migration for people who have the previous version installed?

Maybe not an issue since we haven't done a release since adapters.xxx was introduced.

Reply:
Yeah, that was my thought. Since we never released, just roll with it.